plugins/modules/rds_global_cluster_info.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023 Ansible Project
+# Copyright (c) 2023 Gomathi Selvi Srinivasan (@GomathiselviS)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+module: rds_global_cluster_info
+version_added: 6.6.0
```

Comment:
Should it be `7.0.0`?

Reply:
Yes, I think so. The next release should be 7.0.0 with all the migration PRs. Am I right?